collect_app/src/main/java/org/odk/collect/android/mainmenu/MainMenuButton.kt

```diff
@@ -46,5 +59,18 @@ class MainMenuButton(context: Context, attrs: AttributeSet?) : FrameLayout(conte
         } else {
             number.toString()
         }
+
+        @ExperimentalBadgeUtils
+        if (highlightable) {
+            if (number > 0) {
+                BadgeUtils.attachBadgeDrawable(badge, binding.icon)
+                binding.name.typeface = Typeface.DEFAULT_BOLD
```

Comment:
I think this should probably be `binding.name.setTypeface(binding.name.typeface, Typeface.BOLD)` so that we retain the font being used. Currently, it just so happens that the default and our theme font are the same.

Reply:
Done.